website/docs/index.html.markdown

```diff
@@ -1,108 +1,88 @@
 ---
 layout: "google"
-page_title: "Provider: Google Cloud"
-sidebar_current: "docs-google-index"
+page_title: "Provider: Google Cloud Platform"
+sidebar_current: "docs-google-provider-x"
 description: |-
-  The Google Cloud provider is used to interact with Google Cloud services. The provider needs to be configured with the proper credentials before it can be used.
+   The Google provider is used to configure your Google Cloud Platform infrastructure
 ---
 
-# Google Cloud Provider
+# Google Cloud Platform Provider
 
-The Google Cloud provider is used to interact with
-[Google Cloud services](https://cloud.google.com/). The provider needs
-to be configured with the proper credentials before it can be used.
+The Google provider is used to configure your [Google Cloud Platform](https://cloud.google.com/) infrastructure. 
+See the [Getting Started](/docs/providers/google/getting_started.html) page for an introduction to using the provider. See the [provider reference](/docs/providers/google/provider_reference.html) for advanced
+details on authenticating or otherwise configuring the provider.
```

Comment:
I do kind of think that we should have some sort of super bare bones example here so people can get an idea of what they're looking at without having to click on a link